hs_core/tasks.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import absolute_import
+
+import os
+import sys
+import traceback
+import zipfile
+import logging
+import time
+
+from celery import shared_task
+
+from hs_core.hydroshare import utils
+from hs_core.models import BaseResource
+
+
+@shared_task
+def add_zip_file_contents_to_resource(pk, zip_file_path):
+    zfile = None
+    try:
+        logger = logging.getLogger('django')
```

Comment:
Given our current logging configuration, using `logging.getLogger(__name__)` results in the messages not being written to the log.  I propose to keep 'django' for now, and we can audit and fix logging in a future issue.